superset-frontend/spec/helpers/theming.ts

```diff
@@ -51,5 +51,5 @@ export function styledShallow(
       theme: supersetTheme,
       ...options?.wrappingComponentProps,
     },
-  });
+  }).dive();
```

Comment:
Should this live here or in the tests using this wrapper? I see it called here: https://github.com/apache/incubator-superset/pull/10649/files#diff-7dabcc63260bfc89250effddae3e73d5R49

are both necessary?

Reply:
Before moving the `dive()` into the helper, That instance had `dive().dive()` happening on it. I can't think of a time when we wouldn't want to `dive()` at least once, so I added it into the helper to reduce/reuse/recycle.

Reply:
ah, so dive only goes 1 component deep, makes sense